src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -975,6 +975,29 @@ public class ShowHideConsoleMenuItemConverter : IValueConverter
         #endregion
     }
 
+    /// <summary>
+    /// Obsolete class of ShowHidePreviewBubblesConverter
+    /// Using of this class will produce compile warnings
+    /// TODO: To be removed in Dynamo 2.0
+    /// </summary>
+    [System.Obsolete("This class is obsolete, string converter for Preview Bubble Display Option only returns single value now. ")]
+    public class ShowHidePreviewBubblesConverter : IValueConverter
+    {
+        #region IValueConverter Members
+
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return Resources.DynamoViewSettingsMenuShowPreviewBubbles;
```

Comment:
I am hesitate to bring the deleted resource string back, so I am informing developer that this converter from now on till 2.0 will return single value only. We can also mark that using this converter will cause compile errors. Open for opinions.